events/events-team/events-lead.md

```diff
@@ -1,33 +1,84 @@
 # Events Lead Handbook
 
+The Event Lead is responsible for acting as a chair for the event, carrying out
+the strategic vision, and being the glue needed to execute on the event.
+
+- [Overview](#overview)
+- [Skills and Qualifications](#skills-and-qualifications)
+- [Activities](#activities)
+- [Scheduling and Communication Property Access](#scheduling-and-communication-property-access)
+
 ## Overview
 
-The Event Lead is responsible for acting as a chair for the event, carrying out the strategic vision, and being the glue needed to execute on the event.
+The Event Lead(s) is responsible for determining which core roles need to be
+filled at the start of planning for each event. After initial core team meetings,
+each lead will determine how to best staff for that function and fill in other
+roles. Each role will ideally have a shadow for succession planning. Those who
+have been a part of a previous event get first dibs on lead roles to maintain
+consistency.
 
-The Event Lead(s) is responsible for determining which core roles need to be filled at the start of planning for each event. After initial core team meetings, each lead will determine how to best staff for that function and fill in other roles. Each role will ideally have a shadow for succession planning. Those who have been a part of a previous event get first dibs on lead roles to maintain consistency.
+Time Commitment:
+- 1-8 hours per week leading up to the event
 
 ## Skills and Qualifications
 
-The Event Lead will have lead at least one role during a previous Contributor Summit. The shadow is signing up to commit to lead a future event within the next 12 months.
+The Event Lead must have lead at least one role during a previous Contributor
+Summit. The shadow is signing up to commit to lead a future event within the
+next 12 months.
+
+- Demonstrate empathy
+- Ability to work in both a high level capacity with regard to event planning as
+  well as being detailed oriented when reviewing specific roles.
+- Be comfortable with delegation, team building, gap analysis, and dealing with
+  conflict
+- Have good communication skills
+- Be able to be impartial and objective
+
+## Activities
+
+- Ensure the overall event is a great experience for all contributors
+- Recruit a core team (see [best-practices.md]) and help recruit other volunteers
+  as necessary
+- Create an event strategy with the Content Lead(s) and consider having a social
+  event (decide if social needs it's own lead)
+- Provide venue ideas and suggestions to CNCF; do a walk through if possible
+- Organize and lead N cadence update meetings
```

Comment:
I think we can make this more clear. Maybe "Organize and lead weekly update meetings"